eclair-core/src/test/scala/fr/acinq/eclair/db/AuditDbSpec.scala

```diff
@@ -37,12 +38,18 @@ import scala.util.Random
 
 class AuditDbSpec extends AnyFunSuite {
 
+  import fr.acinq.eclair.TestDatabases.forAllDbs
+
   val ZERO_UUID: UUID = UUID.fromString("00000000-0000-0000-0000-000000000000")
 
-  test("init database 2 times in a row") {
-    forAllDbs { dbs =>
-      val db1 = dbs.audit
-      val db2 = dbs.audit
+  test("init sqlite 2 times in a row") {
```

Comment:
nit: rename "init database 2 times in a row"? Same in other DB test files

Reply:
Done with 2d717ed.